wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -764,16 +764,16 @@ def icon(name=None, classname=None, title=None, wrapped=False, class_name=None):
 
         warn(
             (
-                "Icon template tag `class_name` has been renamed to `classname`, please adopt the new usage instead.",
-                f'Replace `{{% icon ... class_name="{class_name}" %}}` with `{{% icon ... classname="{class_name}" %}}`',
```

Comment:
I assume this was mistaken with the `checks` classes that accept `hint` as the second argument 😄